app/src/main/java/campus/tech/kakao/map/data/remote/ConfigService.kt

```diff
@@ -0,0 +1,40 @@
+package campus.tech.kakao.map.data.remote
+
+import android.util.Log
+import campus.tech.kakao.map.data.vo.Config
+import com.google.firebase.Firebase
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import com.google.firebase.remoteconfig.remoteConfig
+import com.google.firebase.remoteconfig.remoteConfigSettings
+import kotlinx.coroutines.tasks.await
+
+class ConfigService {
+
+    private val remoteConfig: FirebaseRemoteConfig = Firebase.remoteConfig
+    private val configSettings = remoteConfigSettings {
+        minimumFetchIntervalInSeconds = 0 // 개발용
+    }
+
+    init {
+        remoteConfig.setConfigSettingsAsync(configSettings)
+    }
+
+    suspend fun getConfig(): Config {
+        remoteConfig.fetchAndActivate().await()
+        Log.d("config", Thread.currentThread().name)
+        return Config(
+            remoteConfig.getString(SERVICE_STATE),
+            remoteConfig.getString(SERVICE_MESSAGE)
+        )
+    }
+
+    companion object {
+        private const val SERVICE_STATE = "serviceState"
+        private const val SERVICE_MESSAGE = "serviceMessage"
+    }
+
+    object ServiceState {
+        const val ON_SERVICE = "ON_SERVICE"
+    }
```

Comment:
이 부분은 enum class를 활용해볼 수도 있습니다. 프로젝트 상황에 맞게 적절히 선택하시면 될 것 같아요!
```suggestion
    enum class  ServiceState {
        ON_SERVICE
    }
    // enum명 그대로 string으로 접근하고 싶을 때
    val onService: String = ServiceState.ON_SERVICE.name
```